Tools/WebKitTestRunner/WebNotificationProvider.cpp

```diff
@@ -163,6 +163,22 @@ WKDictionaryRef WebNotificationProvider::notificationPermissions()
     return m_permissions.get();
 }
 
+// If the state is not stored in m_permissions, return false. Otherwise, return true and set
+// permissionIsGranted based on if the state is granted or denied.
+bool WebNotificationProvider::getPermissionState(WKSecurityOriginRef securityOrigin, bool& permissionIsGranted)
```

Comment:
out-parameters are so old fashioned. In new code, I'd prefer if we returned an `std::optional<bool>` instead:
```
std::optional<bool> WebNotificationProvider::permissionState(WKSecurityOriginRef securityOrigin)
```

Also, per coding style, WebKit getters should not start with the prefix "get".

Reply:
Fixed!